src/components/user-research-banner/user-research-banner.yaml

```diff
@@ -39,7 +39,3 @@ examples:
     url: 'https://www.example.com/user-research'
     language: en
     hideCloseButton: true
-
-visualRegressionTests:
-  backstopScenarioOptions:
-    delay: 2000
```

Comment:
not sure why this was needed, maybe the hover and focus again  - but havent had any flakey tests since removing so going to try leaving it off 🤞